sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
 ## 1.2.0-beta.2 (Unreleased)
-- Reverted change in 1.2.0-beta.1 which moved @rollup/plugin-json from devDependencies to dependencies
+- Reverted change in 1.2.0-beta.1 which moved @rollup/plugin-json from devDependencies to dependencies.
+- `DefaultAzureCredential` now by default shows the Device Code message on the console. This can still be overwritten with a custom behavior by specifying a function as the third parameter, `userPromptCallback`.
```

Comment:
Shouldn't the default be to not prompt, especially since this shipped? This seems like a breaking, *blocking* behavior change as it waits for user input. In .NET, we have a boolean parameter to opt into interactive prompts (or go full-on pick-and-choose with an options class).

Please bear in mind, I'm only seeing this description, though. Perhaps I'm misinterpreting it having not seen the code review, but it really does seem to indicate you'll block on user input by default.

Reply:
Thank you for your time, Heath!

The changes in my PR only add a logger by default. They don't affect the blocking behavior.